add-on/TA-Demisto/default/app.conf

```diff
@@ -11,7 +11,7 @@ label = Demisto Add-on for Splunk
 [launcher]
 author = Demisto Inc
 description = This application provides an alert action to create an incident in Demisto.
-version = 2.0.3
+version = 2.0.4
 
```

Comment:
Do we build and CI already?

Reply:
@orius123 no. Don't think we will for this one